### PR TITLE
Update transaction types panel name

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/7398
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: grafana-tidb-dashboard.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ To understand the key metrics displayed on the TiDB dashboard, check the followi
...
     - Load Safepoint OPS: the number of times that `Safepoint` is loaded. `Safepoint` is to ensure that the data before `Safepoint` is not read when the transaction reads data, thus ensuring data safety. The data before `Safepoint` might be cleaned up by the GC
     - Pessimistic Statement Retry OPS: the number of retry attempts for pessimistic statements. When the statement tries to add lock, it might encounter a write conflict. At this time, the statement will acquire a new snapshot and add lock again
-    - Async Commit Transaction Counter: the number of transactions that have async commit enabled, which has two statuses: successful and failed
+    - Transaction Types Per Seconds: the number of transactions per second using the two-phase commit (2PC), async commit, and one-phase commit (1PC) mechanisms, including both success and failure numbers
```
Comment (shichun-0415) [inline grafana-tidb-dashboard.md:78]:
```suggestion
    - Transaction Types Per Seconds: the number of transactions committed per second using the two-phase commit (2PC), async commit, and one-phase commit (1PC) mechanisms, including both success and failure transactions
```

Comment (shichun-0415) [reply grafana-tidb-dashboard.md:78]: Per Seconds 看起来语法不太对，但是原文好像就是这么写的，不知道这是固定参数，还是说可以修改。

Comment (sticnarf) [reply grafana-tidb-dashboard.md:78]: 嗯，的确不对。这个是监控面板上的 title，什么时候统一检查修改一下这种问题吧…

**Review by shichun-0415 — APPROVED** [review]

**Review by sticnarf — APPROVED** [review]

**Pull request merged**